.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @adamhicks
```

Comment:
What about [creating teams](https://docs.github.com/en/organizations/organizing-members-into-teams) and adding users to the team? That way the code owner can be set to a specific team (even if you are the only person in that team). Eventually more people can be added to the teams that can provide approvals, and the teams can be split per repo. This can provide a bit more fine-grained control and management via the Github UI v.s editing this file for every change.
E.G this could then be something like  `* @luno/lu-reviewers`

Reply:
I think we can use teams to manage a set of reviewers for all repos, for instance we could have lead reviewers that are added to all reviews across the organisation.

I would rather manage repo specific reviewers like this because I think managing it through the github UI is just obfuscating the people who are needed to review.

Reply:
I do generally like teams over individuals, so that we don't have to edit CODEOWNERS as people join/leave. **But** I don't anticipate needing to have a CODEOWNERS for all repos - just the ones where we want to stray from "any org member can approve"

So for me personally I'm happy to have a single CODEOWNER requirement for this repo, since it's (currently) a one-off